jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -45,11 +51,24 @@
 @Singleton
 @Startup
 @ConcurrencyManagement(ConcurrencyManagementType.CONTAINER)
+@TransactionManagement(TransactionManagementType.BEAN)
 @Lock(LockType.READ)
 public class EJBTimerScheduler {
 
 	private static final Logger logger = LoggerFactory.getLogger(EJBTimerScheduler.class);
 
+    static private enum TimerExceptionPolicy {
+        RETRY,
+        IGNORE,
+        PLATFORM
```

Comment:
do we really need to have the policy here? what would be the need for IGNORE or PLATFORM? And what PLATFORM means ...

Reply:
PLATFORM is dependant. The spec says that it will retry at least one, but that is app to app server to decide. in wildfly case it will retry once right after failure.
IGNORE is just it fails, don't try anymore.

Reply:
I'd remove the IGNORE one and default to PLATFORM as you did already

Reply:
remove IGNORE